projects/media-viewer/src/lib/viewers/image-viewer/image-viewer.component.ts

```diff
@@ -98,19 +90,12 @@ export class ImageViewerComponent implements OnChanges {
     }
   }
 
-  setImageStyles() {
-    this.zoomStyle = `scale(${this.zoom})`;
-    this.rotationStyle = `rotate(${this.rotation}deg)`;
-  }
-
   setZoomValue(zoomValue) {
-    return new Promise((resolve) => {
```

Comment:
does this still update the zoom value in the toolbar? because that's why the promise was added

Reply:
I re-introduced that work-around